cmd/root.go

```diff
@@ -60,4 +61,16 @@ func init() {
 
 	rootCmd.PersistentFlags().BoolVarP(&GlobalVerbose, "verbose", "v", false, "verbose output")
 
+	newTag, isNew, err := util.GetNewerReleaseTag(VERSION)
+	if err != nil {
+		// The error is intentionally not being handled because we don't want
+		// to stop the execution of the program because of this failure
+		log.Infof("Error checking if newer ocdev release is available: %v", err)
+	}
+	if isNew {
+		fmt.Printf("\nA newer version of ocdev (version: " + fmt.Sprint(newTag) + ") is available.\n\n" +
```

Comment:
can we please remove empty lines?  I don't think we need to have extra empty line after each line of text.

But on the other hand, I would add one extra empty line after the end.

Reply:
ack, removed